compiler/ballerina-lang/src/main/resources/grammar/BallerinaParser.g4

```diff
@@ -44,12 +44,7 @@ serviceDefinition
     ;
 
 serviceBody
-    :   LEFT_BRACE serviceBodyMember* RIGHT_BRACE
-    ;
-
-serviceBodyMember
-    :   objectFieldDefinition
-    |   objectFunctionDefinition
+    :   LEFT_BRACE objectFunctionDefinition* RIGHT_BRACE
```

Comment:
We seem to have another deviation. Resource functions cannot have visibility qualifiers. Let's fix that separately as discussed offline. Created https://github.com/ballerina-platform/ballerina-lang/issues/17607 to track that.